pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -122,10 +129,24 @@ func (c *cyclonedxParser) addPackages(cdxBom *cdx.BOM) {
 			CPEs:     []string{comp.CPE},
 			NodeData: *assembler.NewObjectMetadata(c.doc.SourceInformation),
 		}
-		c.rootPackage.depPackages = append(c.rootPackage.depPackages, parentPackages{
+		parentPkg := parentPackages{
```

Comment:
hmm can we rename this to `package` or `component` node, or something, just parentPackage doesn't seem to match the description anymore?

Reply:
done!